BreakOut/spinn_breakout/python_models/breakout_machine_vertex.py

```diff
@@ -13,21 +13,19 @@
 # ----------------------------------------------------------------------------
 # BreakoutMachineVertex
 # ----------------------------------------------------------------------------
-class BreakoutMachineVertex(MachineVertex,
-                            ProvidesProvenanceDataFromMachineImpl):
+class BreakoutMachineVertex(MachineVertex):
 
     _BREAKOUT_REGIONS = Enum(
         value="_BREAKOUT_REGIONS",
         names=[('SYSTEM', 0),
-               ('BREAKOUT', 1),
-               ('PROVENANCE', 2)])
+               ('BREAKOUT', 1)])
```

Comment:
It should, but I haven't done it since breakout.c has it's own Enum for representing regions and it seems it's using it for some initialization (by retrieving the memory address of the region). Was not completely sure how to deal with it so I took an executive decision to ignore it for now.